docs/email/templates.html

```diff
@@ -0,0 +1,298 @@
+---
+layout: page-nomenu
+title: Email Templates
+description: A few starter templates that allow us to create consistently designed, properly rendered emails.
+---
+<section class="stacks-section">
+    {% header {element: 'h2', text: 'Templates'} %}
+    <div class="grid gsx gs16 grid__allcells6 sm:fd-column">
+        <div class="grid--cell">
+            <p class="stacks-copy">Stacks provides 3 starter templates. Each template is explained below, detailing how and when to use them.</p>
+            <ol class="stacks-list">
+                <li><a href="#short-transactional">Short Transactional</a></li>
+                <li><a href="#long-transactional">Long Transactional</a></li>
+                <li><a href="#promotional">Promotional</a></li>
+            </ol>
+        </div>
+        <div class="grid--cell">
+            <div class="grid ai-start gs4 s-notice s-notice__info">
+                <div class="grid--cell mr8">{% svg /resources/svg-icons/Help.svg width="18" height="18" aria-hidden="true" class="svg-icon iconHelpt" %}</div>
+                <div class="grid--cell lh-lg">
+                    <strong>Note:</strong> If an email inherits from a master template (eg. <code class="stacks-code bg-powder-200">_Stacks-Layout.Html.cshtml</code>), only the middle part of the template is needed (the <code class="stacks-code bg-powder-200">&lt;head&gt;</code>, logo, and footer are automatically included). Each template includes <code class="stacks-code bg-powder-200">&lt;!-- EMAIL BODY : BEGIN --&gt;</code> and <code class="stacks-code bg-powder-200">&lt;!-- EMAIL BODY : END --&gt;</code> markers indicating this middle&nbsp;section.
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="stacks-section">
+    {% header {element: 'h2', text: 'Short Transactional'} %}
+    <p class="stacks-copy">
+        A starting point for simple transactional emails with a single message.
+    </p>
+    <div class="grid grid__allcells6 sm:fd-column jc-space-between gs16 gsx">
+        <div class="grid--cell s-card mb16">
```

Comment:
Given our current discussions around `s-card`, should we remove the `s-card` here and reserve it only for clickable link areas?